Source/WebCore/platform/graphics/FontDescription.h

```diff
@@ -172,8 +160,9 @@ class FontDescription {
     AtomString m_locale;
     AtomString m_specifiedLocale;
 
+    std::optional<FontSizeAdjust> m_sizeAdjust;
```

Comment:
Shouldn't the optional be moved inside the FontSizeAdjust? We're not saying "there's optionally a font size adjust", but instead we're saying "there is always a font size adjust, and the value of it may be none"

So, perhaps something like `using FontSizeAdjust = std::optional<InnerFontSizeAdjust>`?